TCP_UDP_server.go

```diff
@@ -0,0 +1,137 @@
+package main
+
+import (
+	"log"
+	"bytes"
+	"fmt"
+	"io"
+	"sync"
+	"net"
+)
+var routinecount sync.WaitGroup
+var waitForgoroutine sync.WaitGroup
+var message string
+
+func main()  {
+	waitForgoroutine.Add(1)
+	go deployTcpServer()
+	deployUdpserver()
+	waitForgoroutine.Wait()
+
+}
+
+func deployTcpServer()  {
+
+	defer waitForgoroutine.Done()
+	// listens for connection
+	listens,error:=net.Listen("tcp","localhost:5002")
+	fmt.Println("Server is listening: ",listens.Addr())
+	if error!=nil{
+		log.Fatal(error)
+	}
+	// listens for connection
+	for{
+		connection,error:=listens.Accept()
```

Comment:
Try to adopt go conventions so user `err` instead of `error`

Reply:
ok i am gonna edit it